sapi/cli/php_cli_server.c

```diff
@@ -348,18 +348,34 @@ static void append_http_status_line(smart_str *buffer, int protocol_version, int
 	smart_str_appendl_ex(buffer, "\r\n", 2, persistent);
 } /* }}} */
 
-static void append_essential_headers(smart_str* buffer, php_cli_server_client *client, int persistent) /* {{{ */
+static void append_essential_headers(smart_str* buffer, php_cli_server_client *client, int persistent, sapi_headers_struct *sapi_headers) /* {{{ */
 {
 	char *val;
 	struct timeval tv = {0};
+	sapi_header_struct *h;
```

Comment:
Style nits: please declare the variable at the same line as the assignment at line 360. We use C99 nowadays in PHP.
Similarly, declare the pos variable inside the `if (sapi_headers != NULL) {` branch.